### PR TITLE
Add `Hooking into mouse events`

### PR DESCRIPTION
**Pull request by GlaceCoding**
It is not obvious to look on prototypes part to find mouse hook function then why I add in hooks part with an extra picture

image

**old version:**
~~[image](https://user-images.githubusercontent.com/92152391/147276050-8d3fd693-36ef-4d37-9fdd-25556d88bac6.png)~~

~~[image](https://user-images.githubusercontent.com/92152391/147276070-7cb3891f-7fa0-4187-b0b4-9fee57ce45b1.png)~~

### CODE REVIEW

File: libs/minilibx/hooks.md [hunk 1]
```diff
@@ -69,6 +69,44 @@ a key. As you can see, we register a hook function with `mlx_key_hook`. However
...
+  - Scroll down : 5  
+
+</td>
+</tr>
+</table>
+  
+</details>
+
+
+
```
Comment (harm-smits) [inline libs/minilibx/hooks.md]: Can you create a markdown table for this?

File: libs/minilibx/hooks.md [hunk 2]
```diff
@@ -69,6 +69,44 @@ a key. As you can see, we register a hook function with `mlx_key_hook`. However
...
+<table>
+<tr>
+<th align="center">Schema</th>
+<th align="center">Buttons</th>
+</tr>
+<tr>
+<td>
+
+![Mouse buttons](https://user-images.githubusercontent.com/92152391/147275282-90ff2fa2-810d-4c7d-b7bd-0c72d6192d7f.png)
```
Comment (harm-smits) [inline libs/minilibx/hooks.md]: Please put this image in the actual repo and define your source. It needs to be credited accordingly.

**Review by harm-smits — CHANGES_REQUESTED** [review]

File: libs/minilibx/hooks.md (hunk 1 above)
Comment (GlaceCoding) [reply libs/minilibx/hooks.md]: We cannot make a list in markdown table

File: libs/minilibx/hooks.md (hunk 2 above)
Comment (GlaceCoding) [reply libs/minilibx/hooks.md]: I took a picture of a random mouse on google (I think it was take by logitech for commercial use).
image

File: libs/minilibx/hooks.md (hunk 1 above)
Comment (GlaceCoding) [reply libs/minilibx/hooks.md]: @harm-smits I used HTML img with right align instead html table

image

**Comment by harm-smits** [comment]
Just for the sake of having mentioned this, but in older version of MLX this was completely oofed. It didn't work at all hence it was not part of the documentation. Glad to see that that has changed now though ;)

**Pull request merged**